### DISCUSSION TITLE
Issue or not

### DISCUSSION THREAD

**Discussion by akrijnen** [microsoft-search/pnp-modern-search#1507] [Q&A]
Hi everyone, 
I am using version 3.22 of the Web Parts. And I have a remarkable thing going on here:

When I search for a name, eg. Miguel I get 👍 
![image](https://user-images.githubusercontent.com/5722079/141122965-42d388ab-3650-426e-93ea-a55b20ef4db3.png)

But when I search for another word, eg: Test it will give me results.

The queries are identical and the POST to: /_api/search/postquery returns in the first situation a 500 will in the 2nd not. 

The body of the first request has the body:

`
{"request":{"__metadata":{"type":"Microsoft.Office.Server.Search.REST.SearchRequest"},"ClientType":"PnPModernSearch","Properties":{"results":[{"Name":"EnableDynamicGroups","Value":{"BoolVal":true,"QueryPropertyValueTypeIndex":3}},{"Name":"EnableMultiGeoSearch","Value":{"BoolVal":true,"QueryPropertyValueTypeIndex":3}},{"Name":"ClientFunction","Value":{"StrVal":"PnPSearchWebPart","QueryPropertyValueTypeIndex":1}},{"Name":"ListId","Value":{"StrVal":"84cd41b1-5cf2-4a9f-b47c-138e5f2d31ff","QueryPropertyValueTypeIndex":1}},{"Name":"ListItemId","Value":{"StrVal":"2","QueryPropertyValueTypeIndex":1}}]},"Querytext":"miguel","TimeZoneId":13,"EnableQueryRules":true,"QueryTemplate":"{searchTerms}","RowLimit":10,"SelectProperties":{"results":["Title","Path","Created","Filename","SiteLogo","PreviewUrl","PictureThumbnailURL","ServerRedirectedPreviewURL","ServerRedirectedURL","HitHighlightedSummary","FileType","contentclass","ServerRedirectedEmbedURL","DefaultEncodingURL","owstaxidmetadataalltagsinfo","UniqueID","ContentTypeId","Message","Title","Tag","Summary","RelatedItem","ItemID","InternalId","GroupUrl","ActivityType","DisplayAuthor","PictureURL","WorkEmail","BaseOfficeLocation","OriginalPath","FileExtension","IsDocument","IsContainer","IsListItem","HtmlFileType","NormSiteID","NormWebID","NormListID","NormUniqueID","ParentLink","Author","AuthorOWSUSER","SPSiteUrl","SiteTitle","SiteId","WebId","LastModifiedTime"]},"TrimDuplicates":false,"SortList":{"results":[{"Property":"Rank","Direction":1}]},"Culture":1033,"Refiners":"RefinableString00(filter=50000/0/*,sort=frequency/descending),FileType(filter=50000/0/*,sort=frequency/descending),DisplayAuthor(sort=frequency/descending),Department(filter=50000/0/*,sort=frequency/descending),Created(discretize=manual/2020-11-10T13:26:39.352Z/2021-08-10T12:26:39.352Z/2021-10-10T12:26:39.352Z/2021-11-03T13:26:39.351Z/2021-11-09T13:26:39.349Z),BaseOfficeLocation(sort=frequency/descending),ows_tax_idFieldName(sort=frequency/descending)","StartRow":0,"HitHighlightedProperties":{"results":[]},"RefinementFilters":{"results":[]},"ReorderingRules":{"results":[]}}}
`

and the 2nd query:

`{"request":{"__metadata":{"type":"Microsoft.Office.Server.Search.REST.SearchRequest"},"ClientType":"PnPModernSearch","Properties":{"results":[{"Name":"EnableDynamicGroups","Value":{"BoolVal":true,"QueryPropertyValueTypeIndex":3}},{"Name":"EnableMultiGeoSearch","Value":{"BoolVal":true,"QueryPropertyValueTypeIndex":3}},{"Name":"ClientFunction","Value":{"StrVal":"PnPSearchWebPart","QueryPropertyValueTypeIndex":1}},{"Name":"ListId","Value":{"StrVal":"84cd41b1-5cf2-4a9f-b47c-138e5f2d31ff","QueryPropertyValueTypeIndex":1}},{"Name":"ListItemId","Value":{"StrVal":"2","QueryPropertyValueTypeIndex":1}}]},"Querytext":"test","TimeZoneId":13,"EnableQueryRules":true,"QueryTemplate":"{searchTerms}","RowLimit":10,"SelectProperties":{"results":["Title","Path","Created","Filename","SiteLogo","PreviewUrl","PictureThumbnailURL","ServerRedirectedPreviewURL","ServerRedirectedURL","HitHighlightedSummary","FileType","contentclass","ServerRedirectedEmbedURL","DefaultEncodingURL","owstaxidmetadataalltagsinfo","UniqueID","ContentTypeId","Message","Title","Tag","Summary","RelatedItem","ItemID","InternalId","GroupUrl","ActivityType","DisplayAuthor","PictureURL","WorkEmail","BaseOfficeLocation","OriginalPath","FileExtension","IsDocument","IsContainer","IsListItem","HtmlFileType","NormSiteID","NormWebID","NormListID","NormUniqueID","ParentLink","Author","AuthorOWSUSER","SPSiteUrl","SiteTitle","SiteId","WebId","LastModifiedTime"]},"TrimDuplicates":false,"SortList":{"results":[{"Property":"Rank","Direction":1}]},"Culture":1033,"Refiners":"RefinableString00(filter=50000/0/*,sort=frequency/descending),FileType(filter=50000/0/*,sort=frequency/descending),DisplayAuthor(sort=frequency/descending),Department(filter=50000/0/*,sort=frequency/descending),Created(discretize=manual/2020-11-10T13:27:10.647Z/2021-08-10T12:27:10.647Z/2021-10-10T12:27:10.647Z/2021-11-03T13:27:10.646Z/2021-11-09T13:27:10.645Z),BaseOfficeLocation(sort=frequency/descending),ows_tax_idFieldName(sort=frequency/descending)","StartRow":0,"HitHighlightedProperties":{"results":[]},"RefinementFilters":{"results":[]},"ReorderingRules":{"results":[]}}}`

The question is now... is this an MS issue or a WebPart issue? We've tested this our with the regular Web Parts and they do not have these issues.

**Comment by wobba** [comment]
You should replicate this outside of PnP and log a support ticket.

**Comment by sw00sh0r** [comment]
We are also investigating 500 errors related to SharePoint Search.
As far as we analyzed the issue, this has nothing to do with the PnP-Search-Webpart but rather SharePoint Search.
Our error is related to the `FileType` column and we are going to raise a Microsoft support ticket to get the log for the correlation id.
Maybe you should do the same as Wobba suggested.

@akrijnen 
Youre issue might be related to your Refiner for `FileType`:
`FileType(filter=50000/0/*,sort=frequency/descending)`

50000 could be too many results for the Rest-API.

**Reply by akrijnen** [reply]
@sw00sh0r  thanks for your reply! I will investigate the FileType thing. And when that does not get resolved I will create a ticket with MS.
